packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/DynamicZone/AddComponentButton.tsx

```diff
@@ -53,45 +53,19 @@ const StyledAddIcon = styled(PlusCircle)<{ $isOpen?: boolean; $hasError?: boolea
   }
   > path {
     fill: ${({ theme, $hasError }) =>
-      $hasError ? theme.colors.danger600 : theme.colors.neutral600};
+      $hasError ? theme.colors.danger600 : theme.colors.neutral500};
   }
 `;
 
-const AddComponentTitle = styled<TypographyComponent>(Typography)``;
+const AddComponentTitle = styled<TypographyComponent>(Typography)`
+  color: ${({ theme }) => theme.colors.neutral600};
+`;
```

Comment:
Looks like it was useless to have a custom component here. Could you try deleting it and and calling `<Typography textColor="neutral600">` instead where it's used?

Reply:
It works but the style is defined next to the style of the error state. Is it fine?

![CleanShot 2024-12-13 at 16 03 26@2x](https://github.com/user-attachments/assets/5247a201-9b3c-4e90-87d1-80535c21800c)

Reply:
i think that's fine if it works for all cases